src/main.rs

```diff
@@ -36,6 +38,13 @@ fn main() {
                     .help("Just print the hex values, not colour previews")
                     .takes_value(false)
             )
+            .arg(
+                Arg::with_name("output")
+                    .long("output")
```

Comment:
I know `--output` is what I suggested in the original ticket, but I'm not sure it's what I want to live with. To me that argument sounds like it's telling me where to send the output (e.g. to stdout, to stderr, to a file).

I'm wondering about `--format`, but that feels like it's CSV/JSON/TXT/whatever.

I'm actually leaning towards `--notation` as the best name for this concept?